copis/core.py

```diff
@@ -495,15 +515,18 @@ def remove_action(self, index: int) -> Action:
         return action
 
     def clear_action(self) -> None:
+        """Remove all actions"""
```

Comment:
I don't think we should have docstrings here -- it's just documentation for the sake of documentation. Pretty clear that `clear_action` clears actions, very redundant

Reply:
Yeah but it prevents `Pylint` from complaining, so let's keep it. I plan on doing that everywhere it's needed.

Reply:
Ok, fair